e2e-tests/demand-backup-cloud/run

```diff
@@ -47,21 +48,19 @@ main() {
 		run_backup "$cluster" "${backup_name_azure}"
 		run_recovery_check "$cluster" "${backup_name_azure}"
 
+		backup_dest_aws=$(kubectl_bin get pxc-backup "$backup_name_aws" -o jsonpath='{.status.destination}' | sed -e 's/.json$//' | cut -c 6-)
```

Comment:
why are we replacing `get_backup_dest` with the command if we still have the function? maybe I'm missing something...

Reply:
So yep) It's question for discussion. Do you think it is better to safe function and it will look like this 
`backup_dest_aws="$(get_backup_dest "${backup_name_aws}")"
  backup_dest_aws=${backup_name_aws:5} # trim "s3://"`
  
  Or just keep like this: 
 ` $(kubectl_bin get pxc-backup "$backup_name_gcp" -o jsonpath='{.status.destination}' | sed -e 's/.json$//' | cut -c 6-)`
 I don't a lot of motivation add new method.  (If second variant is ok, I delete get_backup_dest method)

Reply:
I'm ok with deleting the function if we don't use it anywhere else.